src-ghc/Pact/Bench.hs

```diff
@@ -205,8 +206,8 @@ benchPures pt dbEnv es = bgroup "pures" $ (`map` es) $
 benchKeySet :: KeySet
 benchKeySet = mkKeySet [PublicKey "benchadmin"] ">"
 
-acctRow :: ObjectMap PactValue
-acctRow = ObjectMap $ M.fromList
+acctRow :: RowData
+acctRow = RowData RDV1 $ fmap pactValueToRowData $ ObjectMap $ M.fromList
```

Comment:
Bikeshed but could also be
`RowData RDV1 . fmap pactValueToRowData . ObjectMap $ M.fromList ...`

Reply:
There's actually some reasons to stick with `$` that @larskuhtz was talking about so not jumping on this :)